biggraphite/drivers/lucene.py

```diff
@@ -72,7 +65,7 @@ def translate_to_lucene_filter(components):
         if gs_index == len(components) - 1:
             # No define length: match on components before the GLOBSTAR
             components.pop()
-            must_list = _build_filters(components)
+            lucene_filter["filter"]["must"] = _build_filters(components)
```

Comment:
add this to must_list, and do `lucene_filter['filer']['must']` later in the code to remove some duplication